projects/GEDKeeper2/GKCore/GKUtils.cs

```diff
@@ -1639,7 +1639,7 @@ public static ExtRect GetFormRect(Form form)
             if (y < 0) y = 0;
             if (w > mw) w = mw;
             if (h > mh) h = mh;

```

Comment:
We have limited width and height to screen's ones. But this window still able to overhang the screen edges! :smiling_imp: What did we want to do with this code?